src/main/groovy/org/bonitasoft/migration/plugin/project/MigrationProject.groovy

```diff
@@ -35,6 +36,8 @@ class MigrationProject implements Plugin<Project> {
             it.name.startsWith(MigrationConstants.MIGRATION_PREFIX)
         }) {
 
+            testProject ->
+
             ext {
```

Comment:
testProject alias has been addded to avoid ambigous default value "it" when defining task "testJar"